package.json

```diff
@@ -32,12 +32,10 @@
   },
   "homepage": "https://www.phpbb.com",
   "devDependencies": {
-    "stylelint": "8.0.0",
-    "stylelint-order": "0.3.0"
-  },
-  "dependencies": {
+    "acorn": "6.1.1",
+    "braces": "2.3.1",
     "del": "^3.0.0",
-    "gulp": "^3.9.1",
+    "gulp": "4.0.0",
```

Comment:
Are you sure you want to fix this to a specific version? Or does npm handle this differently?